scripts/docker-run.sh

```diff
@@ -58,6 +58,7 @@ $DOCKER_RUN --rm \
 	--env DELPHIX_SIGNATURE_URL \
 	--env DELPHIX_SIGNATURE_TOKEN \
 	--env DELPHIX_PLATFORMS \
+	--env DELPHIX_ENTIRE_VERSION \
```

Comment:
I think it might be better to rename this to something more vague, like `DELPHIX_APPLIANCE_VERSION`.. this way, if we ever change the implementation of `delphix-entire`, we likely won't have to change the meaning/name of the variable (i.e. the interface of the build won't have to change).